examples/Demo/FluentUI.Demo.Client/Documentation/Components/Badge/FluentBadge.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Badge
+route: /Badge
+---
+
+# Badge
+
+The `Badge` component is a visual indicator that communicates a status or description of an associated component.
+It uses short text, color, and icons for quick recognition and is placed near the relavant content.
+
+## Best practices
+
+### Badges should not receive focus
+- Badge information should be surfaced as part of the control that it is associated with, because, badges themselves do not receive focus meaning they are not directly accessible by screen readers. If the combination of icon and badge communicates some meaningful information, that information should be surfaced in another way through screenreader or tooltip on the component the badge is associated with.
+
+### Screen Readers
+Badge content is exposed as text, and is treated by screen readers as if it were inline content of the control it is associated with.
+This should provide a reasonable default for most badges that contain plain text, such as the `CounterBadge`.
+
+There are two actions authors should consider taking when using Badge to improve this experience:
+
+1. If the badge contains a custom icon, that icon must be given alternative text with aria-label, unless it is purely presentational:
+   ```
+   <FluentBadge IconLabel="paste" />
+   ```
+
+1. If the text of the badge itself is not sufficient to convey its meaning, it can either be given additional hidden text, or the parent element given an explicit label:
+    ```
+    <FluentButton>
+        Inbox
+        <FluentBadge>6<span class="visuallyHidden"> unread messages</span></FluentBadge>
```

Comment:
Wouldn't it be nice to have a Parameter `HiddenLabel` in the component for this? No? 
I don't like the dev having to use HTML/CSS for simple cases. I feel like we're missing something :-)

Reply:
No, I think that is a bit too much of an edge case to have a special parameter for. Also , it is not part of the Web Component implementation and I think we need to try to stay close to those. If we would add this, you know the next question will be "Can I put the hidden content in front of the actual content?" and then we need to add another parameter, etc.

Reply:
Mmm. Yes. But maybe we can create a FluentHidden component to solve that. If the dev need to write CSS, we're missing something 😇

Reply:
As discussed, we will remove theis example for now. Can add back later once decided if we add the needed classes in CSS for this